### PR TITLE
pt-br: Remove trailing punctuation in headers

### PR DESCRIPTION
**Pull request by queengooborg**
This PR removes trailing punctuation in headers for the Brazilian Portuguese locale.

### CODE REVIEW

File: files/pt-br/learn/forms/how_to_structure_a_web_form/index.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Observe que um widget pode ser aninhado dentro de seu elemento \<label>, assim:
 
 Mesmo nesses casos, entretanto, é considerada a melhor prática definir o atributo 'for' porque algumas tecnologias assistivas não entendem os relacionamentos implícitos entre labels e widgets.
 
-### Labels são clicáveis também !.
+### Labels são clicáveis também !
```
Comment (josielrocha) [inline files/pt-br/learn/forms/how_to_structure_a_web_form/index.md:102]:
```suggestion
### Labels são clicáveis também!
```

**Review by josielrocha — COMMENTED** [review]
Thank you for your help @queengooborg!

Could you please apply the suggestion to remove space before `!`?

Oh, The PR builder is failling too

**Comment by queengooborg** [comment]
Hey @josielrocha, thank you for reviewing these PRs!

Unfortunately, a lot of these PRs are ending up with a failing test due to an unrelated build flaw that has been around since before the flaw was coded (ex. the issue was introduced to the document, the flaw was coded, but the flaw was never fixed afterwards).  Since the entire document is checked, it'll fail if the document has any issue, even if it's unrelated to the changes made.  It's pretty much always the `unsafe_html` flaw, I have noticed.

While it is important to fix the flaw, it isn't something we have to worry about during these formatting PRs.  We can safely ignore these PR test failures for the time being!  (I do plan on fixing the flaws in the near future, however!)

**Review by josielrocha — APPROVED** [review]
LGTM ;)

**Comment by josielrocha** [comment]
> Hey @josielrocha, thank you for reviewing these PRs!
> 
> Unfortunately, a lot of these PRs are ending up with a failing test due to an unrelated build flaw that has been around since before the flaw was coded (ex. the issue was introduced to the document, the flaw was coded, but the flaw was never fixed afterwards). Since the entire document is checked, it'll fail if the document has any issue, even if it's unrelated to the changes made. It's pretty much always the `unsafe_html` flaw, I have noticed.
> 
> While it is important to fix the flaw, it isn't something we have to worry about during these formatting PRs. We can safely ignore these PR test failures for the time being! (I do plan on fixing the flaws in the near future, however!)

Great, thank you @queengooborg for your help!

**Pull request merged**